new/components/nginx/config/rest-location.cloudify

```diff
@@ -1,4 +1,4 @@
-location ~ ^/api/v2/(blueprints|executions|deployments|nodes|events|search|status|provider|api|node-instances|version|evaluate|deployment-modifications|tokens) {
+location ~ ^/api/v2/(blueprints|snapshots|executions|deployments|nodes|events|search|status|provider|api|node-instances|version|evaluate|deployment-modifications|tokens) {
```

Comment:
agreed. it's a bad and misleading term. we are not snapshotting, we are creating backup.